### ISSUE TITLE
Water sensor

### ISSUE THREAD

**Issue by indi81** [pyalarmdotcom/alarmdotcom#201]
**Which device would you like us to support?**
https://www.securitashome.se/product.html/vattendetektor-2

**Where does this device appear in the Alarm.com mobile app?**
Water sensor ("Water" in Swedish app)

**What types of actions does this device support?**
none

**Which provider do you use?**
Securitas Home

**Include pyalarmdotcomajax Server Output**

====[ Water Sensor ]====

```json
{
   "data":[
      {
         "id":"104954853-14",
         "type":"devices/water-sensor",
         "attributes":{
            "deviceType":null,
            "openClosedStatus":null,
            "state":5,
            "displayStateText":"Dry",
            "isNormalActivityMonitoringEligible":false,
            "hasNormalActivityMonitoring":false,
            "sensorNamingFormat":3,
            "hasState":true,
            "managedDeviceType":19,
            "canBeRenamed":true,
            "canBeDeleted":false,
            "canAccessWebSettings":false,
            "canAccessAppSettings":false,
            "webSettings":1001,
            "canAccessTroubleshootingWizard":false,
            "troubleshootingWizard":null,
            "addDeviceResource":0,
            "canBeAssociatedToVideoDevice":false,
            "associatedCameraDeviceIds":{
               
            },
            "macAddress":"",
            "manufacturer":null,
            "isAssignedToCareReceiver":false,
            "isOAuth":false,
            "isZWave":false,
            "supportsCommandClassBasic":false,
            "isMalfunctioning":false,
            "canBeSaved":true,
            "canChangeDescription":true,
            "description":"Vattenlarm diskho k\u00f6k",
            "deviceModelId":125,
            "canConfirmStateChange":true,
            "canReceiveCommands":false,
            "remoteCommandsEnabled":true,
            "hasPermissionToChangeState":true,
            "deviceIcon":{
               "icon":318
            },
            "batteryLevelNull":null,
            "lowBattery":false,
            "criticalBattery":false
         },
         "relationships":{
            "system":{
               "data":{
                  "id":"14993221",
                  "type":"systems/system"
               }
            },
            "stateInfo":{
               "data":{
                  "id":"104954853-14-2",
                  "type":"devices/state-info"
               }
            }
         }
      },
      {
         "id":"104954853-15",
         "type":"devices/water-sensor",
         "attributes":{
            "deviceType":null,
            "openClosedStatus":null,
            "state":5,
            "displayStateText":"Dry",
            "isNormalActivityMonitoringEligible":false,
            "hasNormalActivityMonitoring":false,
            "sensorNamingFormat":3,
            "hasState":true,
            "managedDeviceType":19,
            "canBeRenamed":true,
            "canBeDeleted":false,
            "canAccessWebSettings":false,
            "canAccessAppSettings":false,
            "webSettings":1001,
            "canAccessTroubleshootingWizard":false,
            "troubleshootingWizard":null,
            "addDeviceResource":0,
            "canBeAssociatedToVideoDevice":false,
            "associatedCameraDeviceIds":{
               
            },
            "macAddress":"",
            "manufacturer":null,
            "isAssignedToCareReceiver":false,
            "isOAuth":false,
            "isZWave":false,
            "supportsCommandClassBasic":false,
            "isMalfunctioning":false,
            "canBeSaved":true,
            "canChangeDescription":true,
            "description":"Vattenlarm tv\u00e4ttmaskin/torktumlare",
            "deviceModelId":125,
            "canConfirmStateChange":true,
            "canReceiveCommands":false,
            "remoteCommandsEnabled":true,
            "hasPermissionToChangeState":true,
            "deviceIcon":{
               "icon":318
            },
            "batteryLevelNull":null,
            "lowBattery":false,
            "criticalBattery":false
         },
         "relationships":{
            "system":{
               "data":{
                  "id":"14993221",
                  "type":"systems/system"
               }
            },
            "stateInfo":{
               "data":{
                  "id":"104954853-15-2",
                  "type":"devices/state-info"
               }
            }
         }
      }
   ],
   "included":[
      
   ],
   "meta":{
      "transformer_version":"1.1"
   }
}
```

**Include Action Endpoints**
none

**Comment by elahd** [comment]
Thanks for providing data. I added water sensor integration to a pre-release branch. Would you mind testing using the instructions at https://github.com/pyalarmdotcom/alarmdotcom/wiki/How-to-Test-&-Debug#install-integration-development-version?

**Comment by elahd** [comment]
Released in 2.1.0.

**Comment by Raul-7-7** [comment]
@indi81 thx for adding this, however after upgrading I am seeing this error: https://github.com/pyalarmdotcom/alarmdotcom/issues/211
